cmd/internal/ec2/add.go

```diff
@@ -63,6 +72,54 @@ func setupAgentForServer(collectionId string) error {
 	return nil
 }
 
+func askToReconfigure() error {
+	printer.Infof("postman-lc-agent is already present as a systemd service\n")
+	printer.Infof("Helpful commands \n Check status: systemctl status postman-lc-agent \n Disable agent: systemctl disable --now postman-lc-agent \n Check Logs: journalctl -fu postman-lc-agent\n Check env file: cat %s \n Check systemd service file: cat %s \n", envFilePath, serviceFilePath)
+	err := survey.AskOne(
+		&survey.Input{
+			Message: "Over-write old apiKey and collectionId with current values ? (y/n)",
+			Help:    "Enter y(default) or n.",
+			// Use the existing value as the default in case we repeat this step
+			Default: "y",
+		},
+		&isReconfigure,
+	)
+	if err != nil {
+		return errors.Wrap(err, "failed to run reconfiguration prompt")
+	}
+	if isReconfigure != "y" {
+		return errors.New("Exiting process because you didn't enter y")
+	}
+	return nil
+}
+
+// Check is systemd service already exists
+func checkReconfiguration() error {
+
+	cmd := exec.Command("systemctl", []string{"is-enabled", "postman-lc-agent"}...)
+	out, err := cmd.CombinedOutput()
+
+	if err != nil {
+		if exitError, ok := err.(*exec.ExitError); ok {
+			exitCode := exitError.ExitCode()
+			if exitCode != 1 {
+				return errors.Wrapf(err, "Received non 1 exitcode for systemctl is-enabled")
```

Comment:
Does this include which exit code it is to help in our own debugging?  (I assume since it's in the original err that it will.)

What does the user see here?  What steps could they take?  If the next step is "contact support with this log output" then we should make that explicit.

Reply:
It's not possible to easily replicate the systemd error codes, so I don't know what the user will see here. 
We are handling these exit codes
- exit code 0: when service is present and enabled
- exit code 1(service is present but disabled)

Docs: https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#Exit%20status